src/gdb.ts

```diff
@@ -55,7 +57,7 @@ class GDBDebugSession extends MI2DebugSession {
 
 	protected override launchRequest(response: DebugProtocol.LaunchResponse, args: LaunchRequestArguments): void {
 		const dbgCommand = args.gdbpath || "gdb";
-		if (this.checkCommand(dbgCommand)) {
+		if (!this.checkCommand(dbgCommand)) {
```

Comment:
how was this not broken before?

Reply:
Yeah, I was quite confused by this too. The check seems to be broken since its introduction in d559072646d27418211a7e1d0605940ef4ec7581. It was not released since then, so I guess it just was not actually tested?

See how `checkCommand` returns true in success and false on failure.

Reply:
cc @GitMensch 

I'll try this out locally

Reply:
i put a pr about this,  #428  https://github.com/WebFreak001/code-debug/pull/428/commits/2869f942a461410004199b4018849f0505766e2f
this is a bug after 0.27.0 in master branch, https://github.com/WebFreak001/code-debug/commit/d559072646d27418211a7e1d0605940ef4ec7581

Reply:
solved, we really should try having some more automated checks that we can run on CI...